pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -2771,3 +2771,8 @@ SELECT num_nulls(a, b), num_nonnulls(a, b) FROM nulls_test
 1  1
 1  1
 2  0
+
+query B
+select (select * from [show crdb_version]) = version()
```

Comment:
[nit] -- keywords like `SELECT` and `FROM` should be in all caps

Reply:
Fixed